src/components/Footer.jsx

```diff
@@ -0,0 +1,39 @@
+import { useContext } from 'react';
+import { StepContext } from '../App';
+import styled, { css } from 'styled-components';
+const Footer = () => {
+  const { currentStep, setCurrentStep } = useContext(StepContext);
+  const nextStep = () => {
+    setCurrentStep({...currentStep, number : ++currentStep.number});
+  }
+  return (
+    <FooterWrap>
+      <Button>이전</Button>
+      <Button type="main" onClick={nextStep}>다음</Button>
+    </FooterWrap>
+  );
+};
+
+export default Footer;
+
+const FooterWrap = styled.footer`
+  width: 100%;
+  height: ${props => props.theme.footerHeight};
+  padding: 8px;
+  position: absolute;
+  bottom: 0;
+  background-color: white;
+  z-index: 100;
+`;
+const Button = styled.button`
+  height: 100%;
+  font-size: 1rem;
+  ${({ type, theme}) => {
+  return css`
+        ${type === "main" ? theme.buttonMain : theme.buttonWhite}
+        width:${type === "main" ? 'calc(82% - 1rem)' : '18%'};
+        min-width: fit-content;
+        margin-left:${type === "main" && '8px'};
+        `
+    }}
```

Comment:
화살표 함수에서 바로 반환하는 경우 중괄호 및 return을 생략할 수 있습니다.

```suggestion
  ${({ type, theme}) =>
     css`
        ${type === "main" ? theme.buttonMain : theme.buttonWhite}
        width:${type === "main" ? 'calc(82% - 1rem)' : '18%'};
        min-width: fit-content;
        margin-left:${type === "main" && '8px'};
        `
    }
```
로 작성할 수 있을 것 같네요. 그리고 vscode에서 [`vscode-styled-components`](https://marketplace.visualstudio.com/items?itemName=styled-components.vscode-styled-components) extension  설치하시면 syntax highlight하고 자동 포맷팅도 지원하니 이용해서 좀 더 깔끔하게 작성하는 것이 좋아보입니다. 저 위에 `width:${`처럼 띄어쓰기가 제대로 적용되지 않는 부분이 생기니깐요.